tests/ResourceSyncingTest.php

```diff
@@ -832,10 +824,9 @@ function migrateUsersTableForTenants(): void
 // Tenant model used for resource syncing setup
 class ResourceTenant extends Tenant
 {
-    public function users()
+    public function users(): BelongsToMany
     {
-        return $this->belongsToMany(CentralUser::class, 'tenant_users', 'tenant_id', 'global_user_id', 'id', 'global_id')
-            ->using(TenantPivot::class);
```

Comment:
Why was the `using()` removed?

Reply:
I'm still looking at why it got removed. I think it's a mistake/leftover when I refactored the `polymorphic` test to a separate test suite. Tests are still passing though. So I'll add it back.

Reply:
I believe this was a mistake. `using` has been added back.